src/__tests__/inspection.test.ts

```diff
@@ -55,14 +55,28 @@ it('reports too few lines with disallowed one-liners.', () => {
   ]);
 });
 
+it('reports no errors on any message when body check is disabled.', () => {
+  const inputs = input.parseInputs('', '', 'true', '', 'true').mustInputs();
+
+  const message =
+    'Change SomeClass to OtherClass\n' +
+    '\n' +
+    'This replaces the SomeClass with OtherClass in all of the module ' +
+    'since Some class was deprecated. This is long message should not ' +
+    'be checked.';
```

Comment:
```suggestion
    'Change SomeClass to OtherClass.';
```

This would have triggered an error without the `skip-body-check` -- could you please double-check?